src/taipy/rest/api/resources/datanode.py

```diff
@@ -125,8 +146,17 @@ class DataNodeList(Resource):
     get:
       tags:
         - api
-      summary: Get a list of datanodes
-      description: Get a list of paginated datanodes
+      summary: Get all data nodes
+      description: |
+        Returns all data nodes.
```

Comment:
'Returns all data nodes.' -> 'All the data nodes are returned as an array of data node items.'